scripts/skills/actives/rf_hold_steady_skill.nut

```diff
@@ -32,6 +32,13 @@ this.rf_hold_steady_skill <- ::inherit("scripts/skills/skill", {
 			text = ::Reforged.Mod.Tooltips.parseString("You and your allies within " + ::MSU.Text.colorPositive(4) " tiles gain the [Holding Steady|Skill+rf_hold_steady_effect] effect for one [turn|Concept.Turn]")
 		});
 
+		ret.push({
+			id = 19,
+			type = "text",
+			icon = "ui/icons/warning.png",
+			text = "Will not affect allies who are [fleeing|Concept.Morale], [stunned|Skill+stunned_effect], or [sleeping|Skill+sleeping_effect]"
```

Comment:
- `Does not affect`
- We try to start warnings at id 20 in Reforged. So this should be 20 and the next one should be 21.